IdentityCore/tests/MSIDAssymetricKeychainGeneratorTests.m

```diff
@@ -94,6 +96,33 @@ - (void)testGenerateKeyPair_whenValidAttributesProvided_andKeyDoesNotExist_shoul
     XCTAssertTrue(publicKeyExists);
 }
 
+- (void)testNSDataExtensions_intByteParser
+{
+    int origonalInt = 256;
+    int origonalIntSize = sizeof(origonalInt);
+    NSData *data = [NSData dataWithBytes: &origonalInt length: origonalIntSize];
+    int convertedInt = [data convertByteRangeToInt:0 numberOfBytesToRead:origonalIntSize];
+    
+    XCTAssertEqual(origonalInt, convertedInt);
+}
+
+- (void)testGenerateKeyPair_generateStkJwk
+{
+    NSString *privateKeyIdentifier = @"com.msal.unittest.privateKey";
+    NSString *publicKeyIdentifier = @"com.msal.unittest.publicKey";
+    
+    MSIDAssymetricKeyLookupAttributes *attr = [MSIDAssymetricKeyLookupAttributes new];
+    attr.privateKeyIdentifier = privateKeyIdentifier;
+    attr.publicKeyIdentifier = publicKeyIdentifier;
+    
+    MSIDAssymetricKeyKeychainGenerator *generator = [self keyGenerator];
+    
+    NSError *error = nil;
+    MSIDAssymetricKeyPair *result = [generator generateKeyPairForAttributes:attr error:&error];
+    
+    XCTAssertNotNil([result generateStkJwk]);
```

Comment:
This test failed in CI, is there an easy way to get the logs out of travis for this?

`MSIDAssymetricKeychainGeneratorTests
  testGenerateKeyPair_generateStkJwk, (([result generateStkJwk]) != nil) failed
  /Users/travis/build/AzureAD/microsoft-authentication-library-common-for-objc/IdentityCore/tests/MSIDAssymetricKeychainGeneratorTests.m:123

    
    XCTAssertNotNil([result generateStkJwk]);
}`